### PR TITLE
fix(oui-tooltip): prevent creating popup if text is empty

### PR DESCRIPTION
**Pull request by JeremyDec**
See context : DTRSD-7165

Signed-off-by: Jérémy De-Cesare <jeremy.de-cesare@corp.ovh.com>

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report yarn-build#1730.0 ✘  
* build & test
  * build ✔ 
  * test ✘

**Comment by ovh-cds** [comment]
CDS Report yarn-build#1730.1 ✘  
* build & test
  * build ✘  
  * test ✘

**Comment by ovh-cds** [comment]
CDS Report yarn-build#1731.0 ✘  
* build & test
  * build ✘  
  * test ✘

File: packages/oui-tooltip/src/tooltip.controller.js [hunk 1]
```diff
@@ -31,14 +33,20 @@ export default class {
                 this.$element.attr("aria-label", this.text);
             }
 
+            const isTextEmptyOrWhitespace = isEmpty(trim(this.text));
```
Comment (FredericEspiau) [inline packages/oui-tooltip/src/tooltip.controller.js]: .Net enough ?

**Comment by FredericEspiau** [comment]
Shouldn't we warn the user instead of just not displaying its tooltip ? It looks like an unexpected side effect

**Comment by ovh-cds** [comment]
CDS Report yarn-build#1731.1 ✘  
* build & test
  * build ✘  
  * test ✘

**Comment by JeremyDec** [comment]
@FredericEspiau, in fact, I was handling this case : https://github.com/ovh/manager/blob/master/packages/manager/apps/web/client/app/domain/general-informations/GENERAL_INFORMATIONS.html#L271.

But you made me think, that the case should be resolve inside the caller scope, and not inside the component, as this fix, looks like hijacking the component itself.

Closing PR

**Pull request closed without merging**